devstack/settings

```diff
@@ -37,11 +37,23 @@ enable_service group-policy
 disable_service tempest
 ENABLED_SERVICES+=,heat,h-api,h-api-cfn,h-api-cw,h-eng
 # NFP services
-enable_service orchestrator,proxy,proxy_agent,base_configurator
+enable_service orchestrator,proxy,proxy_agent,config_orch
 
 # Deployment preferences
 SYSLOG=${SYSLOG:-True}
 
 # Skip exercises by default (can be overridden in local.conf)
 SKIP_EXERCISES=${SKIP_EXERCISES:-volumes,trove,swift,sahara,euca,bundle,boot_from_volume,aggregates,zaqar,client-env,neutron-adv-test,floating_ips,client-args,horizon,sec_groups,gbp_servicechain,gbp_heat}
 
+# Service VM names
+ConfiguratorQcow2Image=/home/stack/service_images/configurator-mitaka-master.qcow2
+VyosQcow2Image=/home/stack/service_images/vyos.qcow2
+HaproxyQcow2Image=/home/stack/service_images/haproxy.qcow2
```

Comment:
The image paths can be given in local.conf, so read them like this,
+ConfiguratorQcow2Image=${ConfiguratorQcow2Image:-/home/stack/service_images/configurator-mitaka-master.qcow2}